### ISSUE TITLE
Build suddenly fails because cannot find igraph

### ISSUE THREAD

**Issue by gluc** [krlmlr/r-appveyor#50]
Thanks a lot for the package!

As of a few days ago, my build for gluc/data.tree suddenly stops working, saying igraph isn't available, though igraph is available. 

My project is rather standard, I would say.

See here for the appveyor file: https://github.com/gluc/data.tree/blob/dev/appveyor.yml 

See here for description: https://github.com/gluc/data.tree/blob/dev/DESCRIPTION

```
+ Rscript -e 'if (!("devtools" %in% rownames(installed.packages()))) q(status=1)'
+ Rscript -e 'library(devtools); library(methods); options(repos=c(CRAN="http://cran.rstudio.com")); install_deps(dependencies = TRUE)'
Skipping 1 unavailable package: igraph
Installing 25 packages: ape, chron, colorspace, data.table, DiagrammeR, dichromat, doParallel, foreach, ggplot2, gridBase, gtable, htmltools, htmlwidgets, httpuv, iterators, labeling, munsell, networkD3, RColorBrewer, reshape2, scales, shiny, treemap, visNetwork, xtable
```

**Comment by gaborcsardi** [comment]
`igraph` is actually not available on r-devel, see e.g. https://cran.rstudio.com/web/packages/igraph/

**Comment by gluc** [comment]
Fair enough. 
The only thing I don't get is why it used to work: https://ci.appveyor.com/project/gluc/data-tree/build/1.0.9

**Comment by gaborcsardi** [comment]
Because r-devel used to have binary igraph, I guess.

I think you can just use r-release instead of r-devel, right @krlmlr ? Or is that not part of the image any more?

**Comment by krlmlr** [comment]
It's still part of the image: https://github.com/krlmlr/r-portable/tree/master/Image. But you need to override `PATH`, as currently it's hard-coded to r-devel: https://github.com/krlmlr/r-appveyor/blob/e2c79142cc29fe0a6d2e2e86448f4c40eb971298/scripts/appveyor-tool.ps1#L75. This is not ideal, a better way would be to use separate images, but don't hold your breath (https://github.com/krlmlr/r-portable/issues/4).

**Comment by krlmlr** [comment]
Do your builds work now?

**Comment by dougmet** [comment]
I have igraph as a Suggests package and had the same problem. Adding:

```
environment:
  R_VERSION: stable
```
to here https://github.com/dougmet/unsumnet/blob/master/appveyor.yml#L16 fixed it for me (see also #53 about release vs stable).

**Comment by gluc** [comment]
Thanks a lot, that did the trick!
